src/test/java/SearchTests.java

```diff
@@ -0,0 +1,68 @@
+import POM.pages.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SearchTests extends BaseTest{
+    @Test
+    public void searchASong(){
+        HomePage homePage = new LoginPage(driver).signIn("nozishka86@gmail.com","te$t$tudent");
+        BasePage basePage = new BasePage(driver).search();
+        SearchPage searchPage = new SearchPage(driver).searchForSong("Pluto");
+
+        Assert.assertTrue(searchPage.plutoFound());
+    }
+
+    @Test
+    public void searchByArtist(){
+        HomePage homePage = new LoginPage(driver).signIn("nozishka86@gmail.com","te$t$tudent");
+        BasePage basePage = new BasePage(driver).search();
```

Comment:
**Please, remove code redundancies!**

**Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )!**

For example

```
 private static final String NAME_SONG = "Pluto";

  @BeforeTest
  public void logIn() {
    HomePage homePage = new LoginPage(driver).signIn("nozishka86@gmail.com", "te$t$tudent");
    BasePage basePage = new BasePage(driver).search();

  }

  @Test
  public void searchASong() {
    SearchPage searchPage = new SearchPage(driver).searchForSong(NAME_SONG);
    Assert.assertTrue(searchPage.plutoFound());
  }

    @Test
    public void clearSearchQuery() {
        SearchPage searchPage = new SearchPage(driver).searchForSong(NAME_SONG).clearSearchInput();
        Assert.assertTrue(searchPage.allClear());
    }
```